packages/graphql/tests/performance/performance.ts

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Driver } from "neo4j-driver";
+import path from "path";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { gql } from "apollo-server";
+import neo4j from "./utils/neo4j";
+import { setupDatabase, cleanDatabase } from "./utils/setup-database";
+import { Neo4jGraphQL } from "../../src";
+import { collectTests } from "./utils/collect-test-files";
+import { ResultsWriter } from "./utils/ResultsWriter";
+import { ResultsDisplay } from "./utils/ResultsDisplay";
+import { TestRunner } from "./utils/TestRunner";
+
+let driver: Driver;
+
+const typeDefs = gql`
```

Comment:
I would say this should be an external file!

Reply:
You mean a `.graphql` file? We can do, although we would need to move the testing graphql files, to avoid confusion